assets/canary/daemonset.yaml

```diff
@@ -0,0 +1,24 @@
+# Hello Openshift Ingress Canary daemonset
+# Specific values are set at runtime
+kind: DaemonSet
+apiVersion: apps/v1
+# name and namespace are set at runtime.
+spec:
+  progressDeadlineSeconds: 600
+  template:
+    spec:
+      priorityClassName: system-cluster-critical
```

Comment:
`priorityClassName: system-cluster-critical` is debatable.  Is this a deliberate choice, or copy and paste?

Reply:
copy paste artifact. Ill drop the `priorityClassName` field for now.